src/otx/tools/templates/detection/detection/cspdarknet_yolox_l/template.yaml

```diff
@@ -28,7 +28,7 @@ hyper_parameters:
         default_value: 0.001
         auto_hpo_state: POSSIBLE
       learning_rate_warmup_iters:
-        default_value: 100
+        default_value: 3
```

Comment:
@sungchul2 , it is not epochs. It is iterations. 3 iterations of warm-up is useless. Please, consider higher value

Reply:
I know and I think it is strange setting as well :D
But it is just for following recipes.

Reply:
Do we have also iterations in OTX1.6?

Reply:
I'm not sure but maybe iteration.

Reply:
But this is actually going to be converted to an OTX 2.2 recipe now, so we can use it the way we're using it now, right? (We just need to choose whether we want to use epochs or iter, and it's up to the IAI team to change this key to iter -> epoch.)

Reply:
Yes, let's consider hyperparameters update, especially for warm-up and early stopping in the nearest future.